### PR TITLE
Fixed issue 205

### PR DESCRIPTION
**Pull request by mishravikas**

### CODE REVIEW

**Comment by debugger22** [comment]
Please add tests which was failing earlier. You can directly use the case I pointed in https://github.com/BinPy/BinPy/issues/205.

Use commit message to close issues. See [this](https://help.github.com/articles/closing-issues-via-commit-messages).

**Comment by coder006** [comment]
@mishravikas use comments to document your code so that everyone knows what problem the changed/added code resolves.

File: BinPy/Gates/gates.py [hunk 1]
```diff
@@ -77,6 +77,11 @@ def setInput(self, index, value):
         # because history_active is set to 0 trigger
         # will get called irrespective of the history.
         else:
+            if(self.size>len(value)):
```
Comment (debugger22) [inline BinPy/Gates/gates.py:80]: Put spaces around `>`
Make it to

```
if(self.size > len(value)):
```

By the way, this is incorrect here. `self` is an instance of `GATES` so it doesn't have any `size`.

File: BinPy/Gates/gates.py [hunk 2]
```diff
@@ -77,6 +77,11 @@ def setInput(self, index, value):
...
+                value.reverse()
+                for i in range(self.size-len(value),0,-1):
```
Comment (debugger22) [inline BinPy/Gates/gates.py:82]: Put spaces after `,` and also around operator.
Convert it to:

```
for i in range(self.size - len(value), 0, -1):
```

and again `self.size` is wrong.

**Comment by mishravikas** [comment]
@debugger22 @coder006 thanks for your comments, will make the above mentioned changes

**Pull request closed without merging**